testsupport/has/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
why do we have such a file here now?

Reply:
do you mean file or line? 
the line was added by my version of go (1.17.x)
the file has been there for a long time already - it's a workaround for proxy e2e tests that use the appstudio tier which has a limited list of resources that users can modify.